__tests__/app.test.js

```diff
@@ -1,9 +1,22 @@
+const db = require('../db/connection');
 const endpointsJson = require("../endpoints.json");
-/* Set up your test imports here */
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const request = require('supertest');
+const app = require('../app');
 
-/* Set up your beforeEach & afterAll functions here */
 
-describe("GET /api", () => {
+
+beforeEach( () => {
+  return seed(testData);
+});
+
+afterAll( () => {
+  return db.end();
+});
+
+
+describe.only("GET /api", () => {
```

Comment:
Please remove all .skip's and .only's before pushing your code. We will ask you to resubmit if we see them going forwards as we can't tell if all your tests are passing.